docs/30_getting-started-guide/index.md

```diff
@@ -28,45 +43,45 @@ In this section, we describe the base components and protocols used in the EOSIO
 
 ## Configuration Guides
 
-The configuration guides section will show you how to configure [Nodeos.](../../glossary/index#nodeos) How [Nodeos](../../glossary/index#nodeos) is configured will depend on what your desired usage is. [Nodeos](../../glossary/index#nodeos) configuration controls which [plugins](../../glossary/index#plugin) are used and how those [plugins](../../glossary/index#plugin) are configured. The [plugins](../../glossary/index#plugin) are used to specify [Nodeos](../../glossary/index#nodeos) behaviour for specific purposes. 
+The configuration guides section will show you how to configure [Nodeos.](../glossary/index/#nodeos) How `Nodeos` is configured will depend on what your desired usage is. `Nodeos` configuration controls which [plugins](../glossary/index/#plugin) are used and how those `plugins` are configured. The `plugins` are used to specify `Nodeos` behaviour for specific purposes. 
 
 ### Smart Contracts and blockchain configuration
-The `EOSIO platform` provides a blockchain platform and a key feature of the `EOSIO platform` is it's flexibility. Smart Contracts are part of this flexibility. Many features are implemented using smart contracts, and as smart contracts can be edited and built so you can customise blockchain behaviour. Some examples are consensus, governance and resource models. For more information see [eosio.contracts](https://developers.eos.io/manuals/eosio.contracts/latest/index) and the [bios boot sequence tutorial.](../../80_tutorials/10_bios-boot-sequence.md)    
+The `EOSIO platform` provides a blockchain platform and a key feature of the `EOSIO platform` is it's flexibility. Smart Contracts are part of this flexibility. Many features are implemented using smart contracts, and as smart contracts can be edited and built so you can customise blockchain behaviour. Some examples are consensus, governance and resource models. For more information see [eosio.contracts](https://developers.eos.io/manuals/eosio.contracts/latest/index) and the [bios boot sequence tutorial.](../80_tutorials/10_bios-boot-sequence.md)    
 
 ### Development and Testing
-There are several ways to configure a [Nodeos](../../glossary/index#nodeos) environment for development and testing. Which option to use largely depends on what the project goals are. Some practical options are provided below.
+There are several ways to configure a [Nodeos](../glossary/index/#nodeos) environment for development and testing. Which option to use largely depends on what the project goals are. Some practical options are provided below.
 
 #### Local Single-Node Testnet
```

Comment:
A query -> we discussed using the phrase "Local single node blockchain" (and I guess "Local Multi-Node blockchain") - is now a good time to introduce these?

Reply:
Assuming that this is a group preferred choice over testnet

Reply:
Yes, I had agreed to the following usage:

* Local single node blockchain
* Local multi-node blockchain

I think we need to get consensus from the team and then do a complete sweet of all the instances on the portal. We can do this in a separate PR. I like this suggestion.